microservices.yml

```diff
@@ -48,7 +48,7 @@ commands:
           name: Populate environment variables from config
           command: |
             [ ! -e build-workspace/environment/config.json ] && mkdir -p build-workspace/environment ; cp environment/config.json build-workspace/environment/config.json || true
-            for s in $(cat build-workspace/environment/config.json | jq -r "to_entries|map(\"\(.key)=\(.value|tostring)\")|.[]" ); do
+            for s in $(cat build-workspace/environment/config.json | jq -r "to_entries|map(\"\(.key)=\(\"\\\"\"+.value|tostring+\"\\\"\")\")|.[]" ); do
```

Comment:
side note: I'm realizing now that we can just remove the overhead of the `for` loop and get the same result.

```
cat ./config.json | jq -r "to_entries|map(\"export \(.key)=\(\"\\\"\"+.value|tostring+\"\\\"\")\")|.[]" >> $BASH_ENV
```

Reply:
i will leave it as is for now

created Jira ticket not to lose it: https://mgmdigitalventures.atlassian.net/browse/SRV-1019